src/common/entities/ECourse.ts

```diff
@@ -0,0 +1,24 @@
+import { Prisma } from '@prisma/client';
+
+export namespace ECourse {
+  export const Basic = Prisma.validator<Prisma.subject_courseArgs>()({});
+  export type Basic = Prisma.subject_courseGetPayload<typeof Basic>;
+
+  export const Extended = Prisma.validator<Prisma.subject_courseArgs>()({
+    include: {
+      subject_department: true,
+      subject_course_professors: { include: { professor: true } },
+    },
+  });
+  export type Extended = Prisma.subject_courseGetPayload<typeof Extended>;
+
+  export const Details = Prisma.validator<Prisma.subject_courseArgs>()({
+    include: {
+      subject_department: true,
+      subject_course_professors: { include: { professor: true } },
```

Comment:
```suggestion
      ...Extended.include
```
얘네 사이에 의존성을 만들까요? (확신 없음)